MapboxCoreNavigation/RouteController.swift

```diff
@@ -353,6 +355,11 @@ open class RouteController: NSObject {
     @objc public func suspendLocationUpdates() {
         locationManager.stopUpdatingLocation()
         locationManager.stopUpdatingHeading()
+        
+        // In case simulated navigation is abruptly stopped before completion,
```

Comment:
I think this sentence is missing a few `the`.

Reply:
@frederoni Haha yeah - *facepalm*  (an old habit of eliminating a few prepositions to keep the comments concise 😸) Good looks. Thanks 👍